dataplane/standalone/sai_entrypoint.cc

```diff
@@ -0,0 +1,54 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "translator.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
```

Comment:
Do you have documentation on why this is needed?

Reply:
i think i just need these on includes. It's so that the c++ funcs are callable from c